### PR TITLE
First version of the pr-status 

### LINKED ISSUE #96: Set PR status

### PR DESCRIPTION
**Pull request by eanker**
image

A first version of the pr-status without databinding.

fixes #96

### CODE REVIEW

**Comment by txsmith** [comment]
Looks good! I'll css this a bit more at a later point in time.

File: src/projects/pull-request/pull-request-general-information.html [hunk 1]
```diff
@@ -73,6 +74,9 @@
         border-left: solid 1px var(--secondary-background-color);
         max-width: 150px;
       }
+      pr-status{
```
Comment (TimvdLippe) [inline src/projects/pull-request/pull-request-general-information.html]: This is not needed. It already has `display: block;` in `:host`

File: src/projects/pull-request/pr-status.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          }
+        },
+        status: {
+          type: String,
+          notify: true
+        },
+        pullRequest: Object
+      },
+      observers: ['_resetStatus(pullRequest)'],
+      _resetStatus: function (){
```
Comment (TimvdLippe) [inline src/projects/pull-request/pr-status.html]: `function() {`

**Comment by TimvdLippe** [comment]
LGTM

**Pull request merged**